src/api/ReportManagement.ts

```diff
@@ -67,6 +67,57 @@ export default class ReportManagement extends Common {
             });
     }
 
+    async getTranscriptOfRecords(username: string, exregName: string): Promise<APIResponse<File>> {
+        const params = {} as any;
+
+        let base64UrlExregName = btoa(exregName);
+        base64UrlExregName = base64UrlExregName.replace(/\+/g, "-");
+        base64UrlExregName = base64UrlExregName.replace(/\//g, "_");
+        base64UrlExregName = base64UrlExregName.replace(/=/g, "");
```

Comment:
why is this base64 encoded? The API does not specify this

Reply:
this was just copy&pasted from getCertificateOfEnrollment for now. I will remove it